packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -446,8 +589,39 @@ export default {
     this.analyticsValues = getAnalyticsValues();
   },
   methods: {
+    setHeight(isNotUSAAndCanada) {
+      let height;
+      if (window.innerWidth > 1314) {
+        if (!isNotUSAAndCanada) {
+          setTimeout(function() {
+            height = document.querySelector('.signup__form').getBoundingClientRect().height + 139 + 'px';
```

Comment:
I think we need to find a better way to modify height than using raw `document.querySelector` low-level APIs. This could be done by changing the className with Vue bindings rather than directly manipulating the height value in JS.

Reply:
We can't just add a class to set the popup height, as there are conditions under which the popup height value can change, for example, the user selects a country

Reply:
Is it possible to allow the pop-up to resize naturally based on its content rather than manually setting it?

Reply:
Due to the specific design of this element, I do not see a possible solution to this issue other than changing the height of the screen when the user interacts